views/islandora_usage_stats_collections.views_default.inc

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Implements hook_views_default_views().
+ */
+function islandora_usage_stats_collection_views_default_views() {
```

Comment:
Actually... The code in this function should just be added to https://github.com/discoverygarden/islandora_usage_stats/blob/7.x/views/islandora_usage_stats.views_default.inc, and another entry created in the array.